Sources/SwiftlySearch/SwiftlySearch.swift

```diff
@@ -53,16 +69,23 @@ fileprivate struct SearchBar: UIViewControllerRepresentable {
         
         private var subscription: AnyCancellable?
         
-        init(text: Binding<String>) {
+        init(text: Binding<String>, placeholder: String?, hidesNavigationBarDuringPresentation: Bool, resultContent: (String) -> ResultContent?) {
             self._text = text
-            self.searchController = UISearchController(searchResultsController: nil)
+            
+            let resultView = resultContent(text.wrappedValue)
+            let searchResultController = resultView.map { UIHostingController(rootView: $0) }
+            self.searchController = UISearchController(searchResultsController: searchResultController)
```

Comment:
I have not had a chance to test this out yet, but could you please explain the rationale behind this code? What is the expected use case?

Reply:
The use case is the same as in UIKit. You can use this for displaying search results in a different context than the view below. It can also be used for example for search suggestions. 
I'm not sure, if I understood your question correctly though.

Reply:
I was worried how it would behave if there *wasn't* a result view, but having tested it it indeed works as I wanted it to.